07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,54 @@
+* {
+    font-family: sans-serif;
+}
+h1 {
+    padding-left: 5%;
+}
+p, a {
+    font-size: 14px;
+}
```

Comment:
when setting this page up, wireframe it with percentages to get it to be more exact. right now, there's more space on the right than the left. you can fix this by maybe increasing the margin/padding for the items on the left.